pulp_smash/utils.py

```diff
@@ -75,7 +75,7 @@ def get_sha256_checksum(url):
     return _CHECKSUM_CACHE[url]
 
 
-def http_get(url, **kwargs):
+def http_get(url, verify=False, **kwargs):
```

Comment:
Since this method doesn't accept ```cfg``` as an argument, it doesn't use the smash config settings, so when it's used with Pulp URLs and doesn't like the SSL, problems happen.

Reply:
Can we make this default to None and take the value from pulp-smash settings "content" role if unset?
This function is meant to be used in tests, and i doubt that any test can make an educated guess about the state of SSL.

Reply:
I'd feel a little weird about doing that, because the content role is specific to Pulp's configuration, but ```http_get()``` is primarily used on non-Pulp resources, like downloading content from external repos to compare the file checksum against Pulp.  That's probably why it never used the ```cfg``` to begin with.